index.html

```diff
@@ -414,359 +342,358 @@ <h2>
-the applicable operations outlined in Section <a href="#methods"></a>.
-      </p>
+        <dt>
+DID Controllers
+        </dt>
+        <dd>
+The <a>controller</a> of a <a>DID</a> is the entity (person or organization)
+that has the capability&mdash;as defined by a <a>DID method</a>&mdash;to make
+changes to a <a>DID document</a>. This capability is typically asserted by the
+control of a set of cryptographic keys, though it may also be asserted via
+other mechanisms. Note that a DID may have more than one controller, and the
+controller(s) may include the <a>DID subject</a>.
+        </dd>
 
-    </section>
+        <dt>
+Verifiable Data Registries
+        </dt>
+        <dd>
+In order to be resolvable to <a>DID documents</a>, <a>DIDs</a> are typically
+recorded on an underlying system or network of some kind. Regardless of the
+specific technology used, any such system that supports recording DIDs and
+returning <a>DID documents</a> is called a <a>verifiable data registry</a>.
+Examples include distributed ledgers, decentralized file systems, databases of
+any kind, peer-to-peer networks, and other forms of trusted data storage.
+        </dd>
 
-    <section>
-      <h2>DID Resolvers</h2>
+        <dt>
+DID documents
+        </dt>
+        <dd>
+<a>DID documents</a> contain metadata associated with a <a>DID</a>. They
+typically express verification methods (such as public keys) and services
+relevant to interactions with the DID subject. A DID document is serialized
+according to a particular syntax (see <a href="#core-representations"></a>).
+The <a>DID</a> itself is the value of the `id` property. The generic properties
+supported in a DID document are specified in <a href="#core-properties"></a>.
+The properties present in a DID document may be updated according to the
+applicable operations outlined in <a href="#methods"></a>.
+        </dd>
 
-      <p>
-A <a>DID resolver</a> is a software and/or hardware component that takes a
-<a>DID</a> (and associated options) as input and produces a <a>conforming
-DID document</a> (and associated metadata) as output in a process
-called <a>DID resolution</a>.
-      </p>
-      <p>
-The inputs and outputs of the <a>DID resolution</a> process are defined in
-<a href="#did-resolution"></a>. Additional considerations for implementing a
-<a>DID resolver</a> are discussed in [[?DID-RESOLUTION]].
-      </p>
-    </section>
+        <dt>
+DID Methods
+        </dt>
+        <dd>
+<a>DID methods</a> are the mechanism by which a particular type of <a>DID</a>
+and its associated <a>DID document</a> are created, resolved, updated, and
+deactivated using a particular <a>verifiable data registry</a>. <a>DID methods</a>
+are defined using separate DID method specifications (see <a href="#methods"></a>).
+        </dd>
 
-    <section>
-      <h2>DID Dereferencers</h2>
+        <p class="note">
+Conceptually, the relationship between this specification and a <a>DID method</a>
```

Comment:
I like this description of DID methods with good analogies. Good work making this relationship between this spec and method specs clear!